SamplesDashboard/SamplesDashboard/Services/AzureSdkService.cs

```diff
@@ -1,99 +0,0 @@
-﻿// ------------------------------------------------------------------------------
```

Comment:
Does this mean that we are no longer displaying the Azure SDK version on the UI when we can?

Reply:
Yes it does. I couldn't understand why we had this service. It looks like it was reporting the version of a NuGet package that was used by an SDK that was ALSO used by the Azure.NET SDK, but what is the point of that? I see the issue where it was requested (https://github.com/microsoftgraph/msgraph-samples-dashboard/issues/82) but it's still not clear what the value of this information actually is.

Reply:
The Graph SDKs have an overall goal to align with the Azure SDKs, my understanding of #82 was to provide an easy means of comparison between the two SDKs possibly also in relation to feature/dependency parity. 
@darrelmiller could also comment on this further.

Reply:
I'll check with Maisa, and can reintroduce this feature if it is still needed/desired.

Reply:
No worries. I guess we can do this in a separate PR. Yes?

Reply:
Yes. Maisa is doing some follow up on this topic.